testrtc.py

```diff
@@ -1,27 +1,70 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 #
 # Copyright 2014 Google Inc. All Rights Reserved.
 
-"""WebRTC Test
+"""WebRTC
 
-This module serves the WebRTC Test Page.
+This module serves the WebRTC troubleshooter page.
```

Comment:
"""
This module handles log file upload and download, deletes log files from the
    blobstore (only allowed via a cron job) at a set interval and also creates
    a 10Mb downloadble file useful for bandwidth benchmarking.
"""